.azure-pipelines/linux-conda-CI.yml

```diff
@@ -16,11 +16,14 @@ jobs:
       Python27:
```

Comment:
Remove other python configuration since it was not tested with nightly build.

Reply:
Removed py2.7 python configuration. Should I include python 3.5?